test_programs/fuzzing_failure/comparison/src/main.nr

```diff
@@ -0,0 +1,7 @@
+// This program will fail when there is a specific difference between the two fields
+#[fuzz]
+fn comparison(a: Field, b: Field) {
+    let c = b + 0xffffffffffff0000000000;
+    let d = c - 0xeeeeeeeeeeeeeeeee;
+    assert((a !=d) | (b == 0) | (a == 0));
```

Comment:
It would destroy the compactness of some of these programs, but we should probably have the formatter run through these files (it must be added in `test_programs/format.sh`)

Reply:
Done